### PR TITLE
Botocore test case

### PR DESCRIPTION
**Pull request by mskanth972**
**Is this a bug fix or adding new feature?**

**What is this PR about? / Why do we need it?**
This PR tests whether the botocore is installed properly in driver

**What testing is done?**

### CODE REVIEW

File: test/e2e/e2e.go [hunk 1]
```diff
@@ -229,6 +230,18 @@ var _ = ginkgo.Describe("[efs-csi] EFS CSI", func() {
 
 	f := framework.NewDefaultFramework("efs")
 
+	ginkgo.It("Validating driver installation", func() {
```
Comment (RyanStan) [inline test/e2e/e2e.go:233]: If there's not a specific reason to have this in a separate Gingko Context than the other tests, than I think we should move this under the existing `ginkgo.Context(storageframework.GetDriverNameWithFeatureTags(driver), func()`

Then, we would have something like 
`ginkgo.It("Validating botocore installation")`

Also, you might want to double check the alignment, I think this block should be indented one more time

**Review by RyanStan — CHANGES_REQUESTED** [review]

File: test/e2e/e2e.go (hunk 1 above)
Comment (mskanth972) [reply test/e2e/e2e.go:233]: `ginkgo.It("Validating driver installation")` is kept as generic. So that many packages can be validated in future.

**Pull request closed without merging**